deployment/persistent_chains/testnet-2.0/testnet-2.0-launch.md

```diff
@@ -0,0 +1,132 @@
+# TestNet 2.0 Launch Guide
+
+This document is a step-by-step guide for DCL TestNet 2.0 launch ceremony and describes in details the following
+stages:
+
+*   node environment preparation
+*   node bootstrap logic
+*   (optional) validator node registration and maintenance
+
+Please see [Running a DCLedger Node](../../../docs/running-node.md) to get the general understanding
+of a node setup logic along with requirements for the hardware and operating system.
+
+## Preparation
+
+**Note** The following steps are expected to be done **before** the ceremony.
+
+1.  ensure a DCL user is in sudoers list (required only for the ceremony):
+    *   Note. by default `ubuntu` user is expected as a running user for the DCL service.
+        You can use/create another one if it doesn't work for you. In any case you will need to ensure
+        that the user can do `sudo`.
+2.  login as a DCL user
+3.  (optional) clean the system
+
+    ```bash
+    $ sudo systemctl stop dcld
+    $ sudo rm -f "$(which dcld)"
+    $ rm -rf "$HOME/.dcl"
+    ```
+4.  get the artifacts:
+    *   release artifacts
+
+        ```bash
+        $ curl -L -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v0.6.1/dcld
+        $ curl -L -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v0.6.1/dcld.service
+        $ curl -L -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v0.6.1/run_dcl_node
+        ```
+
+    *   (for non-genesis nodes only) genesis file. **Note** it will become available once the genesis node is up
+
+        ```bash
+        $ curl -L -O https://raw.githubusercontent.com/zigbee-alliance/distributed-compliance-ledger/master/deployment/persistent_chains/testnet-2.0/genesis.json
+        ```
+5.  put `dlcd` binary in a folder listed in `$PATH` (e.g. `/usr/bin/`) and set a proper owner and executable permissions. Example:
+
+    ```bash
+    $ sudo cp -f ./dcld -t /usr/bin
+    $ sudo chown <dcl-user> /usr/bin/dcld
+    $ sudo chmod u+x /usr/bin/dcld
+    # verification
+    $ dcld version
+    ```
+6.  Create keys for a NodeAdmin and/or Trustee accounts
+
+    **IMPORTANT** keep generated data (especially the mnemonic) securely
+
+    ```bash
+    dcld keys add <key-name> 2>&1 | tee <key-name>.dclkey.data
+    ```
+7.  (Optional) Configure the firewall
+
+    *   p2p and RPC (by default: `26656` and `26657` respectively) should be available for TCP connections
```

Comment:
Can we provide a command/script that can make sure that the ports are open? For example, it may take persistent_peers file as an input and check every ip:port.